raster-foundry/raster-foundry
# Add scopes datamodel

## Overview

This PR adds a datamodel for scopes and a bunch of canned policies to make our lives easier.

### Checklist


### Notes

I left out permissions around project layers on purpose. I also left out permissions for the user endpoints, since all of those endpoints are things that users read about/do to themselves anyway.
I maybe missed more stuff, but I think that it's the case that everything anyone would want to do in the main application is covered here. Regular users explicitly don't have the ability to edit uploads, since that's a system process.

## Testing Instructions

- run tests
- ensure that correct permission relationships are captured by ascending user -> team admin -> organization admin -> platform admin scale

Closes #5259
